set.mm

```diff
@@ -658729,6 +658808,91 @@ not even needed (it can be any class).  (Contributed by Glauco
+       Glauco Siliprandi, 5-Feb-2022.) $)
+    pnfged $p |- ( ph -> A <_ +oo ) $=
+      ( cxr wcel cpnf cle wbr pnfge syl ) ABDEBFGHCBIJ $.
+  $}
+
+  ${
+    xrnpnfmnf.1 $e |- ( ph -> A e. RR* ) $.
+    xrnpnfmnf.2 $e |- ( ph -> -. A e. RR ) $.
+    xrnpnfmnf.3 $e |- ( ph -> A =/= +oo ) $.
+    $( An extended real that is neither real nor plus infinity, is minus
+       infinity.  (Contributed by Glauco Siliprandi, 5-Feb-2022.) $)
+    xrnpnfmnf $p |- ( ph -> A = -oo ) $=
+      ( cr wcel cmnf wceq wo wn cxr cpnf wne wa jca xrnepnf sylib pm2.53 sylc )
+      ABFGZBHIZJZUAKUBABLGZBMNZOUCAUDUECEPBQRDUAUBST $.
+  $}
+
+  $( An upper set of integers is a subset of the complex numbers.  (Contributed
+     by Glauco Siliprandi, 5-Feb-2022.) $)
+  uzsscn $p |- ( ZZ>= ` M ) C_ CC $=
+    ( cuz cfv cr cc uzssre ax-resscn sstri ) ABCDEAFGH $.
+  ${
+    absimnre.1 $e |- ( ph -> A e. CC ) $.
+    absimnre.2 $e |- ( ph -> -. A e. RR ) $.
+    $( The absolute value of the imaginary part of a non real, complex number,
+       is strictly positive.  (Contributed by Glauco Siliprandi,
+       5-Feb-2022.) $)
+    absimnre $p |- ( ph -> ( abs ` ( Im ` A ) ) e. RR+ ) $=
+      ( cim cfv imcld recnd cc0 cr wcel wceq cc wb reim0b mtbid neqned absrpcld
+      syl ) ABEFZATABCGHATIABJKZTILZDABMKUAUBNCBOSPQR $.
+  $}
+
+  $( The standard topologies on the extended reals and on the complex numbers,
+     coincide when restricted to the reals.  (Contributed by Glauco Siliprandi,
+     5-Feb-2022.) $)
+  xrtgcntopre $p |- ( ( ordTop ` <_ ) |`t RR ) = ( ( TopOpen ` CCfld ) |`t RR )
+                  $=
+    ( cioo crn ctg cfv cle cordt cr crest co ccnfld ctopn xrtgioo tgioo2 eqtr3i
+    eqid ) ABCDEFDGHIZJKDZGHIPPOLQQOMN $.
+  ${
+    absimlere.1 $e |- ( ph -> A e. CC ) $.
+    absimlere.2 $e |- ( ph -> B e. RR ) $.
+    $( The absolute value of the imaginary part of a complex number is a lower
+       bound of the distance to any real number.  (Contributed by Glauco
+       Siliprandi, 5-Feb-2022.) $)
+    absimlere $p |- ( ph -> ( abs ` ( Im ` A ) ) <_ ( abs ` ( B - A ) ) ) $=
+      ( cmin co cim cfv cabs cle wcel wbr recnd subcld absimle syl cc0 imsubd
+      cc reim0d oveq2d imcld subid1d 3eqtrrd fveq2d abssubd 3brtr4d ) ABCFGZHIZ
+      JIZUIJIZBHIZJICBFGJIKAUITLUKULKMABCDACENZOUIPQAUMUJJAUJUMCHIZFGUMRFGUMABC
+      DUNSAUORUMFACEUAUBAUMAUMABDUCNUDUEUFACBUNDUGUH $.
+  $}
+
+  $( The positive reals are a subset of the extended reals.  (Contributed by
+     Glauco Siliprandi, 5-Feb-2022.) $)
+  rpssxr $p |- RR+ C_ RR* $=
+    ( crp cr cxr rpssre ressxr sstri ) ABCDEF $.
+
+  ${
+    min2d.1 $e |- ( ph -> A e. RR ) $.
+    min2d.2 $e |- ( ph -> B e. RR ) $.
+    $( The minimum of two numbers is less than or equal to the second.
+       (Contributed by Glauco Siliprandi, 5-Feb-2022.) $)
+    min2d $p |- ( ph -> if ( A <_ B , A , B ) <_ B ) $=
+      ( cr wcel cle wbr cif min2 syl2anc ) ABFGCFGBCHIBCJCHIDEBCKL $.
+  $}
```

Comment:
can you move this next to min1d ?

Reply:
Moved as suggested (in the latest commit)